samtranslator/translator/translator.py

```diff
@@ -31,6 +31,39 @@ def __init__(self, managed_policy_map, sam_parser, plugins=None):
         self.plugins = plugins
         self.sam_parser = sam_parser
 
+    function_names = dict()
+
+    def _get_function_names(self, resource_dict, parameter_values):
+        if resource_dict.get("Type", "").strip() == 'AWS::Serverless::Function':
+            if resource_dict.get('Properties', "").get('Events', ""):
+                events = list(resource_dict.get('Properties', "").get('Events', "").values())
+                for item in events:
+                    # If the function event type is `Api`then gets the function name and
+                    # adds to the function_names dict with key as the api_name and value as the function_name
+                    if item.get('Type', "") == 'Api' and item.get('Properties', None) and item.get('Properties', "")\
```

Comment:
You don't need ", None) in 'get', that's the default behavior

Reply:
Thank you. I have updated the code without ""/None